server/src/http/unleash_client.rs

```diff
@@ -50,14 +50,18 @@ lazy_static! {
         "client_feature_fetch",
         "Timings for fetching features in milliseconds",
         &["status_code", "app_name", "instance_id"],
-        vec![1.0, 2.0, 5.0, 10.0, 20.0, 50.0, 100.0, 200.0, 500.0, 1000.0, 5000.0]
+        vec![
+            1.0, 2.0, 5.0, 10.0, 20.0, 50.0, 100.0, 200.0, 500.0, 1000.0, 5000.0
+        ]
```

Comment:
Is this cargo fmt changes, or is this your editor @kwasniew ?

Reply:
cargo fmt after rust toolchain update. It did it to the entire codebase but I didn't want to pollute this PR

Reply:
Not worried about this then. :)